docs/source/examples/unbounded.py

```diff
@@ -0,0 +1,20 @@
+from gpkit import Variable, Model
+from gpkit.tools import determine_unbounded_variables
+
+A = Variable("A")
+D = Variable("D")
+F = Variable("F")
+mi = Variable("m_i")
+
+Fs = Variable("Fs", 0.9)
+mb = Variable("m_b", 0.4)
+V = Variable("V", 300)
+
+m = Model(F, [F >= D + V**2,
+              D == V**2*A,
+              V >= mi + mb,
+              Fs <= mi,
+             ])
+# sol = m.solve("mosek")  # returns UNKNOWN
+bounds = determine_unbounded_variables(m, "mosek")
```

Comment:
changing it to 1e-20 (a 10^10 fold increase) does not fix it.

changing it to 1e-10 results in a different error,

```
FAIL: test_unbounded_debugging (gpkit.tests.t_model.TestSP_mosek_cli)
Test nearly-dual-feasible problems
----------------------------------------------------------------------
Traceback (most recent call last):
  File "gpkit/tests/t_model.py", line 360, in test_unbounded_debugging
    self.assertEqual(bounds["sensitive to lower bound"], [x.key])
AssertionError: Lists differ: [] != [x]

Second list contains 1 additional elements.
First extra element 0:
x

- []
+ [x]
?  +
```